doctr/datasets/iiit5k.py

```diff
@@ -67,27 +65,22 @@ def __init__(
             if not os.path.exists(os.path.join(tmp_root, _raw_path)):
                 raise FileNotFoundError(f"unable to locate {os.path.join(tmp_root, _raw_path)}")
 
-            if target_task == "text-detection":
-                # x, y, width, height -> xmin, ymin, xmax, ymax
-                box_targets = [[box[0], box[1], box[0] + box[2], box[1] + box[3]] for box in box_targets]
-
-                if rotated_bbox:
-                    # box_targets: xmin, ymin, xmax, ymax -> x, y, w, h, alpha = 0
-                    box_targets = [
-                        [
-                            (box[0] + box[2]) / 2, (box[1] + box[3]) / 2, box[2] - box[0], box[3] - box[1], 0
-                        ] for box in box_targets
-                    ]
-
-                if len(_raw_label) != len(box_targets):
-                    raise ValueError(f"{_raw_label} and {box_targets} are not same length")
-
-                self.data.append((_raw_path, dict(boxes=np.asarray(
-                    box_targets, dtype=np_dtype), labels=tuple(_raw_label))))
-            elif target_task == "text-recognition":
-                self.data.append((_raw_path, dict(labels=_raw_label)))
-            else:
-                raise ValueError(f"{target_task} is not supported")
+            # x, y, width, height -> xmin, ymin, xmax, ymax
+            box_targets = [[box[0], box[1], box[0] + box[2], box[1] + box[3]] for box in box_targets]
+
+            if rotated_bbox:
+                # box_targets: xmin, ymin, xmax, ymax -> x, y, w, h, alpha = 0
+                box_targets = [
+                    [
+                        (box[0] + box[2]) / 2, (box[1] + box[3]) / 2, box[2] - box[0], box[3] - box[1], 0
+                    ] for box in box_targets
+                ]
+
+            if len(_raw_label) != len(box_targets):
+                raise ValueError(f"{_raw_label} and {box_targets} are not same length")
+
+            self.data.append((_raw_path, dict(boxes=np.asarray(
+                box_targets, dtype=np_dtype), labels=tuple(_raw_label))))
```

Comment:
mind changing this to a list? and adding a comment why there is some type casting needed?